app/views/hub/messages/index.html.erb

```diff
@@ -32,10 +32,31 @@
     <ul class="message-list" data-js="messages-pub-sub" data-client-id="<%=@client.id%>"></ul>
   <% end %>
   <div class="hub-section">
-    <%= form_with model: @outgoing_text_message, url: hub_client_outgoing_text_messages_path(client: @client), local: true, builder: VitaMinFormBuilder, method: "post", html: {class: 'text-message-form'} do |f| %>
-      <%= f.cfa_input_field :body, t(".text_message_form.label") %>
-      <%= f.hidden_field :client_id %>
-      <%= f.submit t(".text_message_form.submit"), class: "button #{"button--disabled" if @client.sms_phone_number.blank?}", disabled: @client.sms_phone_number.blank? %>
+    <div class="communication-preferences">
+      <% if !@client&.intake.email_address.present? && !@client&.intake.sms_phone_number.present? %>
```

Comment:
I think `unless @client&.intake.email_address.present? && @client&.intake.sms_phone_number.present?` would be a little more clear

Reply:
good call! thank you.

Reply:
BUT, just a clarification that the opposite of `if !x && !y` would need to be `unless x || y`